### DISCUSSION TITLE
Integration module don't working

### DISCUSSION THREAD

**Discussion by protecme** [Security-Onion-Solutions/securityonion#11343] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

12

### RAM

64

### Storage for /

2to

### Storage for /nsm

20to

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello, 

Hello, 

I need a little help.
I installed SOC 2.4.10 without any problem, but when I launch the installation of the netflow module via fleet I find the configuration of the elastic agent with the netflow module.
I configure the firewall to push the flow to port 2205
and configure the SOC firewall as follows: 
Firewall : 
customhostgroup0 = IP range of my probe 

Portgroups : 
customhostgroup0 : 
TCP = 2055
UDP = 2055

Role :
Sensor : 
Chain : 
Docker-User :
Hostgroups :
customhostgroup0 
Portgroups = customhostgroup0

And 
Role :
Sensor : 
Chain : 
input : 
Hostgroups : 
customhostgroup0 
Portgroups = customhostgroup0

on netflow module configuration, nothing special to say
left in localhost with port 2205 integrated in "so-grid-nodes_general" policy

when I connect to the probe with ssh and perform a tcpdump host IP-FIREWALL and port 2205
I get no packets.

do you have a way for me to check the logs to see if packets are arriving and can be processed correctly?
it's the same for panws module but in SOC dashboard have many missing on dashboard for elastic-agent.filebeat

thank you very much for your help

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
> I configure the firewall to push the flow to port 2205

> Portgroups :
customhostgroup0 :
TCP = 2055
UDP = 2055

> when I connect to the probe with ssh and perform a tcpdump host IP-FIREWALL and port 2205

So you're sending traffic to port 2205 but you allowed port 2055 through the firewall?

**Comment by protecme** [comment]
thank you for your reply.
it's a error for the tcpdump.
after some research I think the netflow module is not available on version 2.4.

at firewall level I have no problem, but in kibana on the netflow dashboard I have no information.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/37100653/e8443ca8-2cee-408f-af74-f8051d1ad2e5)

can you confirm that this is a problem with the netflow module that will be implemented in the next update?

many thanks

**Reply by 0nnyx** [reply]
Using standalone, I see data/graphs on the netflow dashboard. However, the data is incomplete. To get full data, I need to untick data_stream_dataset:netflow.log. Maybe try that and see if it shows anything. On 2.3, the kibana dashboards were empty unless removing that data source type.

**Reply by weslambert** [reply, marked as answer]
The netflow integration is not supported at this time. It will be supported in the next release. 

https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/dev/salt/elasticfleet/defaults.yaml#L58

I've added a note to the documentation about the currently supported integrations.

https://docs.securityonion.net/en/2.4/elastic-fleet.html#integrations
